tests/rules/test_hadoop_dfs_missing_dash.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from thefuck.rules.hadoop_dfs_missing_dash import match, get_new_command
+from tests.utils import Command
+
+
+@pytest.mark.parametrize('command', [
+    Command(script='./bin/hdfs dfs ls', stderr='ls: Unknown command\nDid you mean -ls?  This command begins with a dash.'),
```

Comment:
I meant more generic in the sense that there are probably other subcommands than '-ls' no? (I don't know `hdfs`)
Anyway yes, it would probably be good to fix any command whose output is of the form "foo: Unknown command\nDid you mean bar?"